ob-elasticsearch.el

```diff
@@ -115,7 +119,9 @@ to do that."
     (beginning-of-buffer)
     (es-mark-request-body)
     (let ((output (when mark-active
-                    (es-org-execute-request (cdr (assoc :jq params))))))
+                    (es-org-execute-request
+                     (cdr (assoc :jq params))
+                     (cdr (assoc :tablify params))))))
```

Comment:
Yeah, since there can be different aggregations at the same level, I need to know the name of the aggregation to pull it out from the hashtable